packages/mdc-density/_functions.scss

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright 2019 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "./variables";
+
+///
+/// Returns component height or size for given density scale and density map.
+///
+/// @param {Number | String} $density-scale - Density scale value for component. Examples: `-3`, `0` or `compact`.
+/// @param {Map} $density-scale-map - Density scale map for the target component.
+///
+/// @example
+///   mdc-density-prop-value(compact, (
+///     default: 36px,
+///     comfortable: 32px,
+///     compact: 24px,
+///   ));
+///   // 24px
+///
+/// @example
+///   mdc-density-prop-value(-2, (
+///     default: 36px,
+///     comfortable: 32px,
+///     compact: 24px,
+///   ));
+///   // 28px
+///
+@function mdc-density-prop-value($density-scale, $density-scale-map) {
+  @if (type-of($density-scale) == "string" and
```

Comment:
Would it make sense to also accept the number values? You could just map them instead of keeping a list
```scss
$mdc-density-supported-scales: (
  default: 0,
  comfortable: -1,
  compact: -2
)
```
Then just do everything with the numeric values

Reply:
`$mdc-density-supported-scales` is universal. & It wouldn't be appropriate if we always map `compact` to `-2` which is different for each component.

Density function do accept numbered values in addition to string / token values. Are you saying that we would need to accept this as parameter to sass function? Please see my below comment.